docs/deploy/aocl_fpga.md

```diff
@@ -71,22 +71,33 @@ Setup
 ```
 - Setup TVM with AOCL and OpenCL enabled.
 
-Emulation
----------
-
-- Run software emulation
+Run Software Emulation
+----------------------
 ```
 export CL_CONTEXT_EMULATOR_DEVICE_INTELFPGA=1
 
 python build.py
 python run.py
 ```
 
-- Run on FPGA devices (not tested)
-    - Change tgt value to "aocl -device=s5_ref" on build.py and run.py
+Run with PCI-E FPGA Cards
+-------------------------
+In build.py and run.py, change tgt value to "aocl".
+```
+python build.py
+python run.py
 ```
-unset CL_CONTEXT_EMULATOR_DEVICE_INTELFPGA
 
+Run with SoC FPGA Boards
```

Comment:
If there are any boards which you already confirmed, it is helpful to describe it (c.f. de1soc, de10-nano, etc.).

Besides, I think the usage of soc fpga similar to the case of Raspberry Pi. It seems useful that soc fpga also can Cross Compilation and RPC as with [Rasberry Pi](https://docs.tvm.ai/tutorials/cross_compilation_and_rpc.html#sphx-glr-tutorials-cross-compilation-and-rpc-py). Can we do it?